src/routes/root.jsx

```diff
@@ -0,0 +1,23 @@
+import { App } from "../App.jsx"
+import {Outlet, Link} from "react-router-dom";
+
+export default function Root() {
+     return (
+         <>
+            <li>
```

Comment:
I think its generally good practise to encase your li elements inside a `ul` (see permitted parents here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li#technical_summary)